src/renderer/components/+workloads-pods/pods.tsx

```diff
@@ -201,5 +242,7 @@ export const Pods = withInjectables<Dependencies>(NonInjectedPods, {
     nodeApi: di.inject(nodeApiInjectable),
     eventStore: di.inject(eventStoreInjectable),
     podStore: di.inject(podStoreInjectable),
+    subscribeToWatchStores: di.inject(subscribeToStoresDisposersInjectable),
+    toggleDetails: di.inject(toggleKubeDetailsPaneInjectable),
```

Comment:
We wouldn't need this business logic in UI if we had `item.toggleDetails()` from before :)